src/Controller/EnMarche/CommitteeDesignation/CandidatureController.php

```diff
@@ -190,13 +194,143 @@ public function finishInvitationStepAction(Committee $committee, UserInterface $
         ]);
     }
 
-    private function createCandidacyForm(CandidacyInterface $candidacy): FormInterface
+    /**
+     * @Route("/mes-invitations", name="_invitation_list", methods={"GET"})
```

Comment:
shouldn't `slug` be present in the route path?

Reply:
yes, but it's into the parent declaration (on the class) 
![image](https://user-images.githubusercontent.com/4582866/97862746-27d1c400-1d06-11eb-9f92-ea4ddc7c303c.png)